xonsh/environ.py

```diff
@@ -847,10 +847,18 @@ def DEFAULT_VARS():
             int,
             str,
             5,
-            "Number of rows to reserve for tab-completions menu if "
+            "Maximum number of rows to display for tab-completions menu if "
             "``$COMPLETIONS_DISPLAY`` is ``single`` or ``multi``. This only affects the "
             "prompt-toolkit shell.",
         ),
+        "RESERVE_SPACE_FOR_MENU": Var(
+            is_int,
+            int,
+            str,
+            DefaultNotGiven,
+            "Additional space to reserve at bottom of screen for tab-completions menu. "
+            "Only affects prompt-toolkit shell.  Deprecated.  See ``$COMPLETIONS_MENU_ROWS``.",
+        ),
```

Comment:
I'm deeply confused -- this adds a new env-var that is already deprecated?

Reply:
@gforsyth, I'm going to stick that quote on my fridge :))